inc/agent.class.php

```diff
@@ -1319,9 +1319,10 @@ protected function enrollByInvitationToken($input) {
          'name'              => 'flyvemdm-' . PluginFlyvemdmCommon::generateUUID(),
          'realname'          => $serial,
          '_profiles_id'      => $config['agent_profiles_id'],
-         'profiles_id'       => $config['agent_profiles_id'],      // Default profile when user logs in
+         'profiles_id'       => $config['agent_profiles_id'], // Default profile when user logs in
          '_entities_id'      => $entityId,
          '_is_recursive'     => 0,
+         '_useremails'       => [$email],
```

Comment:
I disagree: this user account is a machine account: we must not add an email here.

You need to remember that there is a machine account for the agent and a "human" account for the user owning a device. I strongly recommend to not have an email address for machine accounts.